x-pack/plugins/security_solution/server/lib/detection_engine/signals/build_bulk_body.test.ts

```diff
@@ -438,13 +444,20 @@ describe('buildBulkBody', () => {
 
 describe('buildSignalFromSequence', () => {
   test('builds a basic signal from a sequence of building blocks', () => {
-    const blocks = [sampleDocWithAncestors().hits.hits[0], sampleDocWithAncestors().hits.hits[0]];
+    const block1 = sampleDocWithAncestors().hits.hits[0];
+    block1._source.new_key = 'new_key_value';
+    block1._source.new_key2 = 'new_key2_value';
+    const block2 = sampleDocWithAncestors().hits.hits[0];
+    block2._source.new_key = 'new_key_value';
+    const blocks = [block1, block2];
     const ruleSO = sampleRuleSO();
     const signal = buildSignalFromSequence(blocks, ruleSO);
     // Timestamp will potentially always be different so remove it for the test
     // @ts-expect-error
     delete signal['@timestamp'];
-    const expected: Omit<SignalHit, '@timestamp'> = {
+    const expected: Omit<SignalHit, '@timestamp'> & { someKey: string; new_key: string } = {
+      someKey: 'someValue',
```

Comment:
I'm not following where `someKey` came from here...

Reply:
`someKey` is part of the initial object that comes back from `sampleDocWithAncestors()`